proto/buf.yaml

```diff
@@ -1,5 +1,5 @@
 version: v1
-name: buf.build/bufbuild/modules
+name: buf.build/bufbuild/modules-proto
```

Comment:
Why rename this? This name isn't currently take AFAICS.

Reply:
I think `managed-modules` makes more sense as a BSR name - the `-proto` convention is one we use for GitHub repos that only contain buf modules but I haven't seen this used as the name of the module in the BSR.

Reply:
I'll change it to `managed-modules`. I changed it originally because `modules` is reserved: https://github.com/bufbuild/core/blob/887666bc634ec020da87ce07dcb72951b1cf057b/internal/bufd/bufdnameblock/bufdnameblock.go#L186

Reply:
Done